src/main/java/org/gene/myfirstservice/controller/OrderController.java

```diff
@@ -0,0 +1,48 @@
+package org.gene.myfirstservice.controller;
+
+import org.gene.myfirstservice.dto.OrderDto;
+import org.gene.myfirstservice.dto.ProductDto;
+import org.gene.myfirstservice.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+
+    @Autowired
+    private OrderService orderService;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<OrderDto> getOrderById(@PathVariable Long id) {
```

Comment:
if you use @RestController annotation, object will automatically converted to JSON. no need to use ResoponwseEntity object